script/README.md

```diff
@@ -25,8 +25,6 @@ table below is represented in the schema file.
 | description | a description of the build | string | true | false | |
 | maintainers | a list of one or more maintainers | array | true | false | |
 | env | an object (dict) of custom environment variables | object with objects | true | false |  |
-| shell | shell interpreter to use for pre and post run | string | true | false | bash |
-| pre_run | script to run before build | string | true | false | |
+| shell | shell interpreter to use for run section | string | true | false | bash |
```

Comment:
Please confirm pre_ and post_ run were removed from buildtest, and how the functionality will be replaced to insert lines before after run.

Reply:
Yes i confirm, the intent was to remove these keys since they are not implemented in the framework and it was not clear how they will be of any use. The ``run`` translates to full content of the test so running something before and after test could have some benefit but i don't have a need at the moment. 

If you think this would be useful, then should have integrate in the framework and have an example in **tutorials** repo.